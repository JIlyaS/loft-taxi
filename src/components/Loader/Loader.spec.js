import renderer from 'react-test-renderer';

import { Loader } from './';

describe('Loader', () => {
  it('renders correctly snapshot', () => {
      const tree = renderer.create(<Loader />).toJSON();
      expect(tree).toMatchSnapshot();
    });
});