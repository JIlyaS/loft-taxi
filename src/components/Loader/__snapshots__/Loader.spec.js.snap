// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly snapshot 1`] = `
<div
  className="loader"
  data-testid="loader"
>
  <img
    alt="Загрузка..."
    className="loader__img"
    height="70px"
    src="loader.svg"
    width="70px"
  />
</div>
`;
